operations/research-and-development/product/product-management-team-handbook/product-ownership-areas.md

```diff
@@ -5,11 +5,12 @@
 | Channels | Chat, Search, Notifications, Status, User Settings, Onboarding, Mobile | Eric Sethna |
 | Playbooks | Incident Response Colloboration, Checklists, Retrospectives | Ian Tao |
 | Boards | Boards, Tasks, Focalboard | Winson Wu, Chen I-Lim |
+| Calls | Audio & Video Calls | 
 | Apps & Integrations | Plugins, Apps, Apps Marketplace, Apps & Plugin Framework | Aaron Rothschild |
-| Growth | Trials, User Activation, Signup Flows, In-Product Notices, Admin Advisor | Don Hogan |
+| Growth | Product-led growth, Self-serve purchases, Trials, In-product & Email Nurtures, Packaging & Pricing | Don Hogan |
```

Comment:
@wiersgallak Propose keeping `Self-serve purchases` under the Self-Service AOR. Made a proposal in your doc as well.

```suggestion
| Growth | Product-led growth, Trials, In-product & Email Nurtures, Packaging & Pricing, A/B Testing Framework | Don Hogan |
```

cc @dontoisme tagging you in case you want to review and add feedback on AORs from your perspective